specification/search/data-plane/Azure.Search/preview/2025-05-01-preview/searchservice.json

```diff
@@ -0,0 +1,14016 @@
+      "required": [
+        "resourceUri",
+        "modelVersion"
+      ]
+    },
+    "AMLVectorizer": {
+      "type": "object",
+      "x-ms-discriminator-value": "aml",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorSearchVectorizer"
+        }
+      ],
+      "properties": {
+        "amlParameters": {
+          "x-ms-client-name": "AMLParameters",
+          "$ref": "#/definitions/AMLParameters",
+          "description": "Specifies the properties of the AML vectorizer."
+        }
+      },
+      "description": "Specifies an Azure Machine Learning endpoint deployed via the Azure AI Studio Model Catalog for generating the vector embedding of a query string."
+    },
+    "AMLParameters": {
+      "type": "object",
+      "properties": {
+        "uri": {
+          "x-ms-client-name": "ScoringUri",
+          "type": "string",
+          "format": "uri",
+          "x-nullable": true,
+          "description": "(Required for no authentication or key authentication) The scoring URI of the AML service to which the JSON payload will be sent. Only the https URI scheme is allowed."
+        },
+        "key": {
+          "x-ms-client-name": "AuthenticationKey",
+          "type": "string",
+          "x-nullable": true,
+          "description": "(Required for key authentication) The key for the AML service."
+        },
+        "resourceId": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "(Required for token authentication). The Azure Resource Manager resource ID of the AML service. It should be in the format subscriptions/{guid}/resourceGroups/{resource-group-name}/Microsoft.MachineLearningServices/workspaces/{workspace-name}/services/{service_name}."
+        },
+        "timeout": {
+          "type": "string",
+          "format": "duration",
+          "x-nullable": true,
+          "description": "(Optional) When specified, indicates the timeout for the http client making the API call."
+        },
+        "region": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "(Optional for token authentication). The region the AML service is deployed in."
+        },
+        "modelName": {
+          "$ref": "#/definitions/AIStudioModelCatalogName",
+          "description": "The name of the embedding model from the Azure AI Studio Catalog that is deployed at the provided endpoint."
+        }
+      },
+      "required": [
+        "uri"
+      ],
+      "description": "Specifies the properties for connecting to an AML vectorizer."
+    },
+    "AIStudioModelCatalogName": {
+      "type": "string",
+      "enum": [
+        "OpenAI-CLIP-Image-Text-Embeddings-vit-base-patch32",
+        "OpenAI-CLIP-Image-Text-Embeddings-ViT-Large-Patch14-336",
+        "Facebook-DinoV2-Image-Embeddings-ViT-Base",
+        "Facebook-DinoV2-Image-Embeddings-ViT-Giant",
+        "Cohere-embed-v3-english",
+        "Cohere-embed-v
```

Comment:
```suggestion
            "description": "Generate embeddings using an Azure Machine Learning endpoint deployed via the Azure AI Foundry Model Catalog at query time."
```

Reply:
... don't know what GitHub did here, this comment is on the completely wrong line